practice-app/app/views.py

```diff
@@ -59,4 +60,41 @@ def WorldTime():
     else:
         return render_template("worldtime.html")
 
+def get_game_id_from_name(game_name):
+    steam_api_game_list_url = "https://api.steampowered.com/ISteamApps/GetAppList/v2/"
+    response = requests.get(steam_api_game_list_url).json()
+    all_games_list = response["applist"]["apps"]
+    game_name = "".join(game_name.lower().split())
+    for game in all_games_list:
+        if game_name == "".join(game["name"].lower().split()): 
+            return game["appid"]
+    return -1
```

Comment:
I'm not sure if this is the best way to indicate there's been a problem as it's way too manual.
Returning None would be better in terms of code readability.